lunabot_control/scripts/differential_drive_controller.py

```diff
@@ -115,12 +75,12 @@ def _loop(self):
         self._right_drive_pub.publish(right_drive_msg)
 
     def constrain(self, val):
-        val = np.clip(-1, val, 1)  # Clipping speed to not go over 100%
-        return np.int8(val * 127 * self.max_speed_percentage)
+        val = np.clip(-self._max_speed, val, self._max_speed)  # Clipping speed to not go over 100%
```

Comment:
Yeah looking at this context self._max_speed should really just be explicity defined as RPM. If needed the paramater is used elsewhere, it would be better to back-calculate it from the rpm value.